presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2559,6 +2581,43 @@ private static HiveStorageFormat extractHiveStorageFormat(Table table)
         throw new PrestoException(HIVE_UNSUPPORTED_FORMAT, format("Output format %s with SerDe %s is not supported", outputFormat, serde));
     }
 
+    private static String encodePreferredOrderingColumns(List<SortingColumn> preferredOrderingColumns)
+    {
+        return Joiner.on(',').join(preferredOrderingColumns.stream()
```

Comment:
Make "," a const and add a comment to state that comma is not a reserved keyword even with quotes (https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL)